internal/cortex/querier/queryrange/instrumentation.go

```diff
@@ -12,19 +12,24 @@ import (
 	"github.com/weaveworks/common/instrument"
 )
 
+const DAY = 24 * time.Hour
+const QUERYDURATIONBUCKET = "query_range_bucket"
```

Comment:
nit: `QUERY_DURATION_BUCKET` for better readability.

Reply:
I will change to camel case, in golang the naming convention is camel case preferred over snake case.